src/fswAlgorithms/imageProcessing/centerOfBrightness/centerOfBrightness.cpp

```diff
@@ -129,3 +129,43 @@ void CenterOfBrightness::UpdateState(uint64_t CurrentSimNanos)
     return;
 }
 
+/*! Method extracts the bright pixels (above a given threshold) by first grayscaling then bluring image.
+ @return std 2 vector of integers
+ @param image openCV matrix of the input image
+ */
+std::vector<cv::Vec2i> CenterOfBrightness::extractPixels(cv::Mat image)
```

Comment:
:small_red_triangle_down: I would be more expressive to rename this method to something like `extractBrightPixels`

Reply:
Done